NORTRIP_save_road_summary_data_netcdf.f90

```diff
@@ -0,0 +1,668 @@
+        
+        call check(nf90_def_var(ncid, "Studs_he", nf90_float, (/f_dimid,t_dimid/),varid)) 
+        call check(nf90_put_att(ncid,varid, "units", "%")) 
+        call check(nf90_put_att(ncid,varid,"description","Percentage of heavy vehicles with studded tyres"))
+        
+        call check(nf90_def_var(ncid, "Speed_li", nf90_float, (/f_dimid,t_dimid/),varid)) 
+        call check(nf90_put_att(ncid,varid, "units", "km/h")) 
+        call check(nf90_put_att(ncid,varid,"description","Speed of light vehicles")) 
+        !TODO: Why is only the speed of light vehicles included in the summary file?
+        
+        call check(nf90_def_var(ncid, "NOX_Emissions_tot", nf90_float, (/f_dimid,t_dimid/),varid)) 
+        call check(nf90_put_att(ncid,varid, "units", "g/km/h")) 
+        call check(nf90_put_att(ncid,varid,"description","Total emissions of NOX")) 
+
+        call check(nf90_def_var(ncid, "Mass_dust_PM200", nf90_float, (/f_dimid,t_dimid/),varid)) 
+        call check(nf90_put_att(ncid,varid, "units", "g/m2")) !TODO: Check units and description
+        call check(nf90_put_att(ncid,varid,"description","Mass of non-suspendable dust (>200 micrometer) on road"))
+        
+        call check(nf90_def_var(ncid, "Mass_sand_PM200", nf90_float, (/f_dimid,t_dimid/),varid)) 
+        call check(nf90_put_att(ncid,varid, "units", "g/m2")) !TODO: Check units and description
+        call check(nf90_put_att(ncid,varid,"description","Mass of non-suspendable sand (>200 micrometer) on road"))
+    else
+        call check(nf90_def_var(ncid, "runway_name", nf90_char,(/char_dimid,f_dimid/),varid))
+        call check(nf90_put_att(ncid,varid, "description", "Location name and segment for runway"))
+        call check(nf90_put_att(ncid,varid, "long_name", "runway_name_and_segment"))    
+    end if
+
+    
+
+
+
+    call check(nf90_enddef(ncid))
+    
+end subroutine NORTRIP_create_summary_netcdf
+
+subroutine NORTRIP_save_road_summary_data_netcdf
+    
+    use netcdf
+    use NORTRIP_definitions
+    
+    implicit none
+    
+    !LOCAL
+    integer :: varid
+    integer :: ncid 
+    integer :: t_dimid
+    integer :: f_dimid
+    integer :: exists
+    real :: timestamp
+    character(len=4)    :: time_string
+    character(len=12)   :: datetime_string
+    integer             :: datetime_int
+    character(len=256)      :: filename
+    character(len=19) :: datetime
+    integer ro_num
+    real ,dimension(max_time_save) :: st_li
+    real ,dimension(max_time_save) :: st_he
+    real ,dimension(max_time_save) :: fr_hdv
+    integer :: a(num_date_index)
+    real    :: conversion
+    real, dimension(max_time_save) :: water,snow,ice
+    real, parameter :: surface_moisture_cutoff = 0.01
+
+    
+    !Check that path exists after filling in date stamp
+    a=date_data(:,min_time_save)
+    
+    filename =trim(path_outputdata)//trim(filename_outputdata)//'_summary.nc'
```

Comment:
Based on discussion, will try to add the file extension to the file name template in the config file, so that alternative suffixes can be used in the applications that need it.